src/scripts/setup.ts

```
@@ -872,9 +775,9 @@ export async function runSetup(): Promise<void> {
         for (const name of selectedNames) {
           const client = detectedClients.find((c) => c.name === name);
           if (!client) continue;
-          const stopSpinner = showSpinner(`Configuring ${client.displayName}...`);
+          const stopSpinner = showSetupProgress(`Configuring ${client.displayName}...`);
           await new Promise((r) => setTimeout(r, 400));
```

ESLint

[error] Return values from promise executor functions cannot be read.

(no-promise-executor-return)